Backend/StemmingFrequency.md

```diff
@@ -17,25 +19,30 @@ Submit your solutions via email as a zip file (please add '.test' to the extensi
 
 At NICE Digital Services, we use Elasticsearch. Elasticsearch indexes content to provide relevant search results for a query. One aspect of indexing is stemming. Stemming is the process for reducing inflected (or sometimes derived) words to their stem, base or root form. For example the stemmed form of 'classes' is 'class'.
 
-Please write code that processes the [input text](#input-tect) to determine the frequency of the stemmed words given in the [acceptance criteria](#acceptance-criteria). Your solution should cope with other input texts containing those terms.
+Please write code that processes the some input text to determine the frequency of the stemmed words within it.  Please see the acceptance criteria below. Your solution should cope with other input texts containing those terms.
 
-### Input text
-
-*Friends are friendlier friendlies that are friendly and classify the friendly classification class. Flowery flowers flow through following the flower flows.*
 
 ### Acceptance criteria
 
-- frequency("following") = 1
-- frequency("flow") = 2
-- frequency("classification") = 1
-- frequency("class") = 1
-- frequency("flower") = 3
-- frequency("friend") = 4
-- frequency("friendly") = 4
-- frequency("classes") = 1
+#### Input text
+
+*"Friends are friendlier friendlies that are friendly and classify the friendly classification class. Flowery flowers flow through following the flower flows."*
+
+Given the input text above, when asked for the following words, return the associated frequency count as specified below
+
+| Word             | Frequency |
```

Comment:
I've done this to leave it open to interpretation what the method signature should be